Script/AtomicEditor/ui/frames/menus/MainFrameMenu.ts

```diff
@@ -306,6 +316,8 @@ var editItems = {
     "Format Code": ["edit format code", StringID.ShortcutBeautify],
     "-5": null,
     "Play": ["edit play", StringID.ShortcutPlay],
+    "Pause/Resume": ["edit pause", StringID.ShortcutPause],
+    "Step": ["edit step", StringID.ShortcutStep],
```

Comment:
Congrats on figuring out all this editor UI stuff including adding custom graphics, hope the skin text needing tabs instead of spaces didn't through anyone for too much of a loop (TurboBadger parser needs some updates in that department), darned cool to see pause and stepping!